systemtest/src/test/java/io/strimzi/systemtest/StrimziUpgradeST.java

```diff
@@ -169,13 +169,15 @@ private void copyModifyApply(File root) {
     }
 
     private void deleteInstalledYamls(File root) {
-        Arrays.stream(Objects.requireNonNull(root.listFiles())).sorted().forEach(f -> {
-            if (f.getName().matches(".*RoleBinding.*")) {
-                cmdKubeClient().deleteContent(TestUtils.changeRoleBindingSubject(f, NAMESPACE));
-            } else {
-                cmdKubeClient().delete(f);
-            }
-        });
+        if (root != null) {
```

Comment:
How come `root` can now be null?

Reply:
Not sure... but tests failed at NPE time by time.